data/json/items/obsolete.json

```diff
@@ -3324,5 +3324,101 @@
     "initial_charges": 3,
     "max_charges": 3,
     "charges_per_use": 1
+  },
+  {
+    "id": "nato_assault_rifle",
+    "copy-from": "rifle_auto",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "IF YOU SEE THIS IT IS A BUG" },
```

Comment:
And I saw it. Probably should add a migration entry for it?

Reply:
If it is possible, could you find out what variant it had? This should only happen if it had no variant (a separate bug) or I missed a migration from one of its variants.

Reply:
I think it's the m4a1 variant, based on the save file.

```
  "typeid": "nato_assault_rifle",
  "item_vars": {
    "dirt": "212",
    "item::mode": "DEFAULT",
    "magazine_converted": "1",
    "shot_counter": "41.000000"
  },
  "owner": "your_followers",
  "invlet": 114,
  "last_temp_check": 0,
  "faults": [
    "fault_gun_dirt"
  ],
  "variant": "m4a1",
```

Reply:
Hhm. It would be reasonable to add a migration at the moment, but I think it is just a bandaid over the bug that it's not being migrated.

Reply:
Oh wait, I wasn't getting notifications for this, so its still getting spawned in new worlds?

Reply:
Just migrations AFAIK